tests/diag.sh

```diff
@@ -284,11 +298,13 @@ case $1 in
 		   echo Shutting down instance 2
 		fi
 		. $srcdir/diag.sh wait-queueempty $2
+		cp rsyslog$2.pid rsyslog$2.pid.save
```

Comment:
That "something else" is rsylog, when it is killed (2 lines below). So this copy should be safe. But still, there is something missing overall ;)